src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -307,7 +311,7 @@ class BookingService(
         expectedDepartureDate = expectedDepartureDate,
         notes = notes,
         booking = booking,
-        createdAt = OffsetDateTime.now(),
+        createdAt = occurredAt,
```

Comment:
The actual UI is still only setup for date - could we perhaps change this to store the date at midnight for now since we're not live and raise a bug to tackle moving both the UI/API to providing time too?

![image](https://user-images.githubusercontent.com/110207532/221628401-18faa777-0a97-43a1-86b5-9978f3631d88.png)

Reply:
yes that sounds like a good plan. thks